catroid/src/org/catrobat/catroid/content/bricks/WhenTouchDownBrick.java

```diff
@@ -52,6 +52,10 @@ public WhenTouchDownBrick() {
 
 	public WhenTouchDownBrick(WhenTouchDownScript script) {
 		this.whenTouchDownScript = script;
+
+		if (script != null && script.isCommentedOut()) {
+			setCommentedOut(true);
+		}
 	}
```

Comment:
Why is the clone method not altered in order to set the cloned brick comment state ?